clients/include/rocblas_arguments.hpp

```diff
@@ -15,11 +15,6 @@
 #include "rocblas.h"
 #include "rocblas_datatype2string.hpp"
 
-typedef enum rocblas_initialization_ {
-    rocblas_initialization_random_int = 111,
-    rocblas_initialization_trig_float = 222,
-} rocblas_initialization;
-
```

Comment:
The `rocblas_initialization` type needs to go into `rocblas_datatype2string.hpp` since that file is included by `rocblas_arguments.hpp` and it needs `rocblas_initialization` to do the conversions.